auction-server/src/auction.rs

```diff
@@ -1428,10 +1546,26 @@ impl ChainStore for ChainStoreSvm {
             }])),
             None => {
                 // not yet confirmed
-                Ok(None)
+                if bids[0].initiation_time + BID_MAXIMUM_LIFE_TIME_SVM < OffsetDateTime::now_utc() {
```

Comment:
this would end up not being confirmed bc it failed on-chain?

Reply:
in general, this branch is a bit confusing, and the documentation in `quote.rs` differs. it would be worth making sure these say the same thing, and also documenting the circumstances under which this branch comes into play here